apps/judicial-system/backend/src/app/formatters/formatters.spec.ts

```diff
@@ -1249,6 +1250,7 @@ describe('formatDefenderCourtDateEmailNotification', () => {
       prosecutor.name,
       prosecutor.institution?.name,
       sessionArrangements,
+      DefenderSubRole.DEFENDANT_DEFENDER,
```

Comment:
To avoid a lot of redundant Contentful strings and functions, I wanted to test adding a defender sub role to control the string concatenation